packages/onboarding/src/utils/flows.ts

```diff
@@ -117,13 +116,12 @@ export const isAnyHostingFlow = ( flowName?: string | null ) => {
 
 export const isAnyMigrationFlow = ( flowName?: string | null ) => {
 	return Boolean(
-		flowName &&
-			[ MIGRATION_FLOW, IMPORT_FOCUSED_FLOW, IMPORT_HOSTED_SITE_FLOW ].includes( flowName )
+		flowName && [ IMPORT_FOCUSED_FLOW, IMPORT_HOSTED_SITE_FLOW ].includes( flowName )
 	);
 };
 
 export const isMigrationFlow = ( flowName: string | null ) => {
```

Comment:
Since we have code already using this method but assuming the old flow name, I think we should create a new one using the `import-focused` value and update all places that are using it.

Reply:
Renamed here: https://github.com/Automattic/wp-calypso/pull/93388/commits/771a735d4b6f37260868d2590591bca419d94387

A related thing, just for context, I also created this issue to investigate these functions later and see if we need to do any other change related to the new flow: https://github.com/Automattic/wp-calypso/issues/93386